frontend/src/app/home/components/map/clickables/clickable-offices/clickable-offices.component.ts

```diff
@@ -19,7 +23,17 @@ export class ClickableOfficesComponent implements OnChanges {
       }
     }
   }
+  whatElement(event: Event){
+    this.mainService.office.forEach(element => {
+      if (element.numberSVG === event.srcElement.id ){
+        this.tooltip = this.tooltipText(element);
+      }
+    })
+  }
 
+  tooltipText(element: any){
```

Comment:
to jest lepsze rozwiązanie ale jeszcze wrzuciłbym to do modelu 
```ts
  tooltipText() {
    return `${this.firstName} ${this.lastName}`;
  }
```
i wtedy wywołujesz 
``` ts
this.tooltip = element.tooltipText();
```